bgpd/bgp_fsm.c

```diff
@@ -1090,6 +1091,7 @@ int bgp_stop(struct peer *peer)
 		/* bgp log-neighbor-changes of neighbor Down */
 		if (bgp_flag_check(peer->bgp, BGP_FLAG_LOG_NEIGHBOR_CHANGES)) {
 			struct vrf *vrf = vrf_lookup_by_id(peer->bgp->vrf_id);
+
```

Comment:
https://github.com/FRRouting/frr/blob/master/doc/developer/workflow.rst
>Whitespace changes in untouched parts of the code are not acceptable in patches that change actual code. To change/fix formatting issues, please create a separate patch that only does formatting changes and nothing else.

Reply:
I had to introduce this new line boz of the error below.
Checkout code: Successful with additional warnings
Report for bgp_fsm.c | 2 issues
===============================================
< WARNING: Missing a blank line after declarations
< #1094: FILE: /tmp/f1-360/bgp_fsm.c:1094:


It is strange that sometimes it gives error on the code that i haven't changed.